File: src/main/java/P3212/ANFBot/Kek.java
package P3212.ANFBot;

import javax.jms.JMSException;
import javax.jms.MessageProducer;
import javax.jms.TextMessage;

public class Kek extends JmsBase {
    private MessageProducer producer = null;

    public Kek()
    {
        super();
        try {
            if (producer != null)
                producer.close();
        } catch (JMSException jmse) {
            recordFailure(jmse);
        }
    }
    @Override
    protected void doAction()
    {
        // Создание текстовых сообщений
        String[] messages = new String[2];
        messages[0] = "Good day";
        messages[1] = "Hello, world!";
        String TEMPL = "Cообщение #%d отправлено";
        try {
            producer = session.createProducer(destination);
            System.out.println("Отправка JMS сообщений\n");
            TextMessage message = null;
            for (int i = 0; i < messages.length; i++) {
                message = session.createTextMessage(messages[i]);
                producer.send(message);
                System.out.println(String.format(TEMPL, (i + 1)));
            }
        } catch (JMSException e) {
            status = -2;
            recordFailure(e);
        }
    }
    public static void main(String[] args)
    {
        new Kek();
        System.exit(status);
    }
}

File: src/main/java/P3212/ANFBot/Kek.kt
package P3212.ANFBot

import com.rabbitmq.jms.admin.RMQConnectionFactory
import javax.jms.JMSException
import javax.jms.MessageProducer
import javax.jms.Session
import javax.jms.TextMessage

object Kek {
    private fun request(msg: String, chatId: Long) {
        val factory = RMQConnectionFactory()
        factory.host = "localhost"
        val con = factory.createConnection()
        val session = con.createSession(false, Session.AUTO_ACKNOWLEDGE)
        val destination = session.createQueue("botQueue")
        val producer = session.createProducer(destination)
        var message = session.createTextMessage(msg)
        producer.send(message)

        val consumer = session.createConsumer(destination)
        val response = consumer.receive(10000)
        println(response)
    }
    @JvmStatic
    fun main(args: Array<String>) {
        request("lel", -1)
    }
}
